Content.Server/AlertLevel/ChangeAlertLevelComponent.cs

```diff
@@ -0,0 +1,18 @@
+using Content.Shared.ChangeAlertLevel;
+using Robust.Shared.Audio;
+
+namespace Content.Server.ChangeAlertLevel
+{
+    /// <summary>
+    ///     Sets an alert level when activated
+    /// </summary>
+    [RegisterComponent]
+    [Access(typeof(ChangeAlertLevelSystem))]
+    public sealed partial class ChangeAlertLevelComponent : SharedChangeAlertLevelComponent
```

Comment:
move this to shared and remove the abstract component, use an abstract empty system for access

i think using abstract components like this is deprecated, and putting it in shared makes predicting it a little easier down the line

Reply:
sure

Reply:
doing this has broken things so terribly

Reply:
why